### ISSUE TITLE
[multiple statement] Error when the query contains multiple statement

### ISSUE THREAD

**Issue by khusamov** [hgourvest/node-firebird#95]
```javascript
Firebird.attach(options, function(err, db) {
 
    if (err)
        throw err;
 
    // db = DATABASE 
    db.query(`
		insert into ELEMENT_GRORDERS_SUB_INFO 
		(ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
		values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 100);
		insert into ELEMENT_GRORDERS_SUB_INFO 
		(ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
		values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 101);
		insert into ELEMENT_GRORDERS_SUB_INFO 
		(ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
		values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 102);
    `, function(err, result) {
        // IMPORTANT: close the connection 
        db.detach();
    });
 
});
```

Error:

```
Unhandled rejection Error: 
Dynamic SQL Error, SQL error code = -104, Token unknown - line 5, column 3, insert
at doCallback (C:\@repositories\element-server\node_modules\node-firebird\lib\index.js:1233:18)
at C:\@repositories\element-server\node_modules\node-firebird\lib\index.js:2928:21
at C:\@repositories\element-server\node_modules\node-firebird\lib\messages.js:151:25
at search (C:\@repositories\element-server\node_modules\node-firebird\lib\messages.js:117:13)
at C:\@repositories\element-server\node_modules\node-firebird\lib\messages.js:54:21
at FSReqWrap.wrapper [as oncomplete] (fs.js:614:17)
```

**Comment by VZiemer** [comment]
Use Firebird execute block

Example:

Firebird.attach(options, function(err, db) {

    if (err)
        throw err;

    // db = DATABASE 
    db.query(`
        execute block as begin insert into ELEMENT_GRORDERS_SUB_INFO 
        (ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
        values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 100);
        insert into ELEMENT_GRORDERS_SUB_INFO 
        (ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
        values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 101);
        insert into ELEMENT_GRORDERS_SUB_INFO 
        (ELEMENT_GRORDERS_SUB_INFOID, GRORDERID, STATUS, WORKSTATIONTYPE) 
        values (gen_id(GEN_ELEMENT_GRORDERS_SUB_INFO, 1), 1, 'hahhaa', 102); end
    `, function(err, result) {
        // IMPORTANT: close the connection 
        db.detach();
    });

});


more documentation in http://www.firebirdsql.org/refdocs/langrefupd20-execblock.html
